build/less/mixins/vertical-rhythm.less

```diff
@@ -0,0 +1,28 @@
+.fontSizeGrid(@step, @size: "xs") {
```

Comment:
There is a newer version of that here:

https://github.com/luehrsenheinrich/chance_ganztag/blob/master/build/less/mixins/verticalRythm.less